recipe/meta.yaml

```diff
@@ -17,7 +17,7 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 2
+  number: 3
   skip: true  # [not linux64 and cuda_compiler_version != "None"]
   skip: true  # [linux64 and cuda_compiler_version == "10.2"]
```

Comment:
Think we can drop this one as conda-forge now only supports CUDA 11.2+ ( https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/1708 )

```suggestion
```